src/main/java/Commands.java

```diff
@@ -0,0 +1,186 @@
+package duke;
+
+import java.util.regex.Pattern;
```

Comment:
It would be beneficial to maintain a consistent import order for better code organization. I recommend placing the 'import java.util.regex.Matcher' statement above the 'import java.util.regex.Pattern' statement to follow the convention of arranging import statements in lexical order.